src/UI/ListView/SummaryItem.elm

```diff
@@ -103,6 +112,39 @@ withSelected selected (SummaryItem prop opt) =
     SummaryItem prop { opt | selected = selected }
 
 
+{-| Adds an icon to the right side of the summary.
+
+    Summary.withIcon (Icon.fix "Problem Solving")
+        someListView
+
+-}
+withIcon : Icon -> SummaryItem msg -> SummaryItem msg
+withIcon icon (SummaryItem prop opt) =
+    SummaryItem prop { opt | icon = Just icon }
+
+
+{-| Adds an icon to the left side of the summary.
+
+    Summary.withBulletIcon (Icon.fixIssues "Problem Solving")
+        someListView
+
+-}
+withBulletIcon : Icon -> SummaryItem msg -> SummaryItem msg
+withBulletIcon icon (SummaryItem prop opt) =
+    SummaryItem prop { opt | bullet = Just icon }
+
+
+{-| Applies a tone to summary's background.
+
+    Summary.withTone Palette.toneWarning
+        someListView
+
+-}
+withTone : Palette.Tone -> SummaryItem msg -> SummaryItem msg
+withTone tone (SummaryItem prop opt) =
+    SummaryItem prop { opt | tone = Just tone }
```

Comment:
why not just take a `Palette.Color`? Also I think "tone" is not a common term for this kind of stuff and I can't tell from just looking at it if it changes the font color or the background, so I suggest going with something like `withBackground` or `withBackgroundColor` which says exactly what it is

Reply:
> why not just take a `Palette.Color`

Mainly because the tone is reused internally with different brightnesses